lib/views/outgoing_mailer/initial_request.text.erb

```diff
@@ -1,6 +1,7 @@
 <%= raw @outgoing_message.body.strip %>
 
 -------------------------------------------------------------------
+<% I18n.with_locale(:fr_BE) do %>
```

Comment:
Hmm, I like that _but_ I can't immediately see how to get the separator text (the shorter `----` line) to appear. Thinking.